spec/controllers/v0/calculator_constants_controller_spec.rb

```diff
@@ -4,10 +4,8 @@
 RSpec.describe V0::CalculatorConstantsController, type: :controller do
   describe 'GET #index' do
     it 'returns calculator constants' do
-      create(:calculator_constant, :float)
-      create(:calculator_constant, :float)
-      create(:calculator_constant, :string)
-      create(:calculator_constant, :string)
+      create(:calculator_constant)
+      create(:calculator_constant)
```

Comment:
NITPICK: 
- Putting this in a loop would help make it seem more deliberate than looking like a duplicate.

`2.times { create(:calculator_constant) }`